core/monitorstorage/storage.go

```diff
@@ -44,54 +45,15 @@ type MonitoringResult struct {
 	Errors []error
 	// Response contains the original mutations API response from the server
 	// in case at least one verification step failed.
-	Response *ktpb.GetMutationsResponse
-}
-
-// Storage is an in-memory store for the monitoring results.
-type Storage struct {
-	store  map[int64]*MonitoringResult
-	latest int64
-}
-
-// New initializes a
-func New() *Storage {
-	return &Storage{
-		store: make(map[int64]*MonitoringResult),
-	}
-}
-
-// Set internally stores the given data as a MonitoringResult which can be
-// retrieved by Get.
-func (s *Storage) Set(epoch int64,
-	seenNanos int64,
-	smr *trillian.SignedMapRoot,
-	response *ktpb.GetMutationsResponse,
-	errorList []error) error {
-	// see if we already processed this epoch:
-	if _, ok := s.store[epoch]; ok {
-		return ErrAlreadyStored
-	}
-	// if not we just store the value:
-	s.store[epoch] = &MonitoringResult{
-		Smr:      smr,
-		Seen:     seenNanos,
-		Response: response,
-		Errors:   errorList,
-	}
-	s.latest = epoch
-	return nil
-}
-
-// Get returns the MonitoringResult for the given epoch. It returns an error
-// if the result does not exist.
-func (s *Storage) Get(epoch int64) (*MonitoringResult, error) {
-	if result, ok := s.store[epoch]; ok {
-		return result, nil
-	}
-	return nil, ErrNotFound
+	Response *pb.GetMutationsResponse
 }
 
-// LatestEpoch is a convenience method to retrieve the latest stored epoch.
-func (s *Storage) LatestEpoch() int64 {
-	return s.latest
+// Interface is the interface for storing monitoring results.
```

Comment:
"Foo is the interface that does bar" seems to be the standard wording for documenting interfaces, so it's unfortunately that when your interface is named Interface you end up with this awkward sentence ...

Reply:
yep